fuzzbench/end-to-end-test-config.yaml

```diff
@@ -0,0 +1,13 @@
+benchmarks:
+    - freetype2-2017
+    - bloaty_fuzz_target
+docker_registry: gcr.io/fuzzbench
+end_to_end_test: true
```

Comment:
can you remove the "end_to_end_test" option from the config? the end to end test should work as a real experiment run, but with an experiment config that only runs very few and short things, e.g., builds just a few images. but it should use the same config options such as fuzzers and benchmarks, so something like "end_to_end_test" should not be part of the experiment config file

Reply:
>can we remove the "end_to_end_test" option from the config?

This is a way to know we are using the specified config.yaml for end-to-end testing.

Again, I think the main point lying in that, **do we need to test the end-to-end testing is using the desired config file.** This comment argument point is based on we actually don't need this test, which means we already know the end-to-end test will use the desired config file.

I think this test included in the end-to-end test is needed to check. This is more like, we need to first sure run on an experiment we desire and then we check experimental behavior.

Otherwise, how to know the modification in compose/e2e-test.yaml is correct? wdyt